game-core/src/main/java/games/strategy/triplea/attachments/UnitAttachment.java

```diff
@@ -3173,12 +3157,14 @@ private void setIsMechanized(@SuppressWarnings("unused") final String s) {}
                 this::setIsAir,
                 this::getIsAir,
                 this::resetIsAir))
-        .put("isMechanized",
+        .put("isMechanized", // kept for map compatibility; remove upon next map-incompatible release
             MutableProperty.ofWriteOnlyString(
-                this::setIsMechanized))
-        .put("isParatroop",
+                s -> {
+                }))
+        .put("isParatroop", // kept for map compatibility; remove upon next map-incompatible release
             MutableProperty.ofWriteOnlyString(
-                this::setIsParatroop))
+                s -> {
+                }))
```

Comment:
I wonder if it would make sense to log a warning here

Reply:
@ron-murhammer You had a concern over similar log verbosity in #4063.  Do you think a) we should be logging deprecated map features in order to identify them for follow-up? and b) if so, what is the appropriate log level to use (noting that we currently have no way to display anything below `INFO` level without a code change; thus, logging will effectively go unnoticed)?